baremetal_minio/service_lookup.py

```diff
@@ -55,6 +55,15 @@ def set_ip_for_host(hostname, ip):
                 f.write(line)
         f.write(f'{ip} {hostname}\n')
 
+def wipe_hosts(hostname):
+    filename = '/etc/hosts'
+    with open(filename, "r") as f:
+        lines = f.readlines()
+
+    with open(filename, "w+") as f:
+        for line in lines:
+            if hostname not in line:
+                f.write(line)
```

Comment:
This is pretty dangerous. I think we should write to a new file and do and move the new version at the end.